src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -87,6 +87,9 @@ public static class ApplicationInsightsTelemetry
         // the session identifier
```

Comment:
@JamesWTruher, your last commit had 2 failures in `PowerShell-CI-macos`
CustomEvent resend check.Should resend startup event if the semaphore says we haven't sent telemetry
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /Users/runner/runners/2.164.7/work/1/s/test/powershell/engine/Basic/Telemetry.Tests.ps1: line 14
14:         $telemetrySent | Should -Be $true
```
CustomEvent resend check.Should resend startup event if the semaphore says we haven't sent telemetry
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /Users/runner/runners/2.164.7/work/1/s/test/powershell/engine/Basic/Telemetry.Tests.ps1: line 14
14:         $telemetrySent | Should -Be $true
```